src/Sylius/Behat/Context/Transform/ProductVariantContext.php

```diff
@@ -41,6 +42,22 @@ public function __construct(ProductRepositoryInterface $productRepository, Produ
     }
 
     /**
+     * @Transform :variant
+     * @Transform /^in variant "["^]+"$/
```

Comment:
`this product is available in variant "XXX"` - what would be implementation of this step? I don't see any use case for variant transformer here because it should create a new variant for "this product".